scripts/migrate-all.sh

```diff
@@ -38,21 +38,23 @@ done
 
 for county in "${COUNTIES[@]}"; do
     aspen_group_id=$(echo "$COUNTY_INFO" | jq -r ".$county".aspen_group_id)
-    if $(echo "$COUNTY_INFO" | jq ".$county | has(\"internal_project_ids\")") = "true"; then
+    if [ "$(echo "$COUNTY_INFO" | jq ".$county | has(\"internal_project_ids\")")" = "true" ]; then
         echo "Importing internal samples for $county..."
         for internal_project_id in $(echo "$COUNTY_INFO" | jq -r ".$county".internal_project_ids[]); do
             aspen-cli db --local import-covidhub-project --rr-project-id "$internal_project_id" --covidhub-db-secret cliahub/cliahub_rds_read_prod  --covidhub-aws-profile biohub --aspen-group-id "$aspen_group_id"
         done
     fi
 
-    if $(echo "$COUNTY_INFO" | jq ".$county | has(\"external_project_id\")") = "true"; then
+    if [ "$(echo "$COUNTY_INFO" | jq ".$county | has(\"external_project_id\")")" = "true" ]; then
         echo "Importing external samples for $county..."
         external_project_id=$(echo "$COUNTY_INFO" | jq -r ".$county".external_project_id)
         aspen-cli db --local import-covidhub-project --rr-project-id "$external_project_id" --covidhub-db-secret cliahub/cliahub_rds_read_prod  --covidhub-aws-profile biohub --aspen-group-id "$aspen_group_id"
     fi
 done
 
 for county in "${COUNTIES[@]}"; do
-    aspen_group_id=$(echo "$COUNTY_INFO" | jq -r ".$county".aspen_group_id)
-    aspen-cli db --local import-covidhub-trees --covidhub-aws-profile biohub --s3-src-prefix s3://covidtracker-datasets/cdph/"$county" --s3-key-prefix /imported/phylo_trees/"$county" --aspen-group-id "$aspen_group_id"
+    if [ "$(echo "$COUNTY_INFO" | jq ".$county.skip_tree_import")" != "true" ]; then
```

Comment:
wait where does `$county.skip_tree_import` come from?

Reply:
> wait where does $county.skip_tree_import come from?

It's set for vrdl.